doc/sphinx/source/install.rst

```diff
@@ -93,6 +95,55 @@ The process documented here will work on Ubuntu\* 16.04 (LTS)
    collaboration diagrams can be found inside the ``/docs/doxygen/`` directory.    
 
 
+CentOS
+------
+
+The process documented here will work on CentOS 7.4.
+
+#. (Optional) Create something like ``/opt/libraries`` and (with sudo), 
+   give ownership of that directory to your user. Creating such a placeholder 
+   can be useful if you'd like to have a local reference for APIs and 
+   documentation, or if you are a developer who wants to experiment with 
+   how to :doc:`../howto/execute` using resources available through the 
+   code base.
+
+   .. code-block:: console
+
+      # mkdir -p /opt/libraries
+      # chown -R username:username /opt/libraries
+
+#. Update the system with :command:`yum` and issue the following commands: 
+   
+   .. code-block:: console
+
+      # yum update
+      # yum -y install zlib-devel
```

Comment:
Any reason this needs to be done in a separate command from the one that follows? (just idly curious)

Reply:
Just the whole long list of commands (3 + lines of wrapping) seemed like a bit much. Visually it's nicer to not have to use horizontal scroll to to see or grab commands